assets/js/components/ProjectEditor.js

```diff
@@ -15,6 +15,8 @@ export function ProjectEditor (projectDescriptionCredits, programId, textFields)
   this.languageSelectorList = $('#edit-language-selector-list')
   this.selectedLanguage = $('#edit-selected-language')
   this.textLoadingSpinner = $('#edit-loading-spinner')
+  this.saveButton = $('#edit-submit-button')
+  this.cancelButton = $('#edit-cancel-button')
```

Comment:
This could now be replaced by the new local variable

https://github.com/danielzhang130/Catroweb/blob/fd3f1dcf5ebd96a94c919bfee062d21292439b93/assets/js/components/ProjectEditor.js#L48

Reply:
Should be updated now